src/applications/edu-benefits/components/EducationWizard.jsx

```diff
@@ -390,22 +390,22 @@ export default class EducationWizard extends React.Component {
                           science, technology, engineering, or math (STEM),{' '}
                           <b>or</b>
                         </li>{' '}
-                        <li>
-                          {' '}
+                        <li className="li-styling">
                           You've already earned a STEM bachelor’s degree and are
-                          pursuing a teaching certification.{' '}
+                          working toward a teaching certification, <b>or</b>
+                        </li>{' '}
+                        <li className="li-styling">
+                          {' '}
+                          You've already earned a STEM bachelor's or graduate
+                          degree and are pursuing a covered clinical training
+                          program for health care professionals. <br />
                           <a
+                            aria-label="See eligible degree programs, opening in new tab"
                             href="https://benefits.va.gov/gibill/docs/fgib/STEM_Program_List.pdf"
                             rel="noopener noreferrer"
                             target="_blank"
-                            onClick={() =>
-                              recordEvent({
-                                event: 'edu-navigation',
-                                'edu-action': 'see-approved-stem-programs',
-                              })
-                            }
```

Comment:
Why is the recording of the event being removed? Did I miss something?

Reply:
I pulled the version of this from https://github.com/department-of-veterans-affairs/vets-website/pull/17642 and then removed Prod Flag

Reply:
Any issues with new content are present in that PR

Reply:
I went ahead and updated my PR https://github.com/department-of-veterans-affairs/vets-website/pull/17642 to have the onclick record event